setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup, find_packages
+
+setup(
+    name="manas-cafa5",
+    version="0.0.0",
+    packages=[],
+    python_requires=">=3.9",
```

Comment:
I'm running everything in python 3.8. Is there a reason you put 3.9 as required?